src/components/ProjectSearch.js

```diff
@@ -164,15 +164,16 @@ class ProjectSearch extends Component {
       {}
     );
 
-    return mergeWith({ ...mappedParams }, filterValues, function (
-      value,
-      srcValue
-    ) {
-      if (value === '') {
-        return srcValue;
+    return mergeWith(
+      { ...mappedParams },
+      filterValues,
+      function (value, srcValue) {
+        if (value === '') {
+          return srcValue;
+        }
+        return value;
```

Comment:
This block could be simplified with an arrow function and a ternary:

```js
return mergeWith(
  { ...mappedParams },
  filterValues,
  (val, src) => val === '' ? src : val
);
```

Reply:
oh nice! I'll fix this as well, I love a good arrow function